File: src/main/java/com/soapdogg/lispInterpreter/printer/AtomNodePrinter.java
package com.soapdogg.lispInterpreter.printer;

import com.soapdogg.lispInterpreter.datamodels.AtomNode;
import lombok.AllArgsConstructor;

@AllArgsConstructor(staticName = "newInstance")
public class AtomNodePrinter {

    String printAtomNode(
        final AtomNode atomNode
    ) {
        return atomNode.getValue();
    }
}

File: src/main/java/com/soapdogg/lispInterpreter/printer/AtomNodePrinter.kt
package com.soapdogg.lispInterpreter.printer

import com.soapdogg.lispInterpreter.datamodels.AtomNode

class AtomNodePrinter {
    fun printAtomNode(
        atomNode: AtomNode
    ): String {
        return atomNode.value
    }
}